src/mscorlib/src/System/Threading/Tasks/Task.cs

```diff
@@ -5549,48 +5549,31 @@ public static Task<TResult> FromException<TResult>(Exception exception)
         /// <summary>Creates a <see cref="Task"/> that's completed due to cancellation with the specified token.</summary>
         /// <param name="cancellationToken">The token with which to complete the task.</param>
         /// <returns>The canceled task.</returns>
-        [FriendAccessAllowed]
```

Comment:
Could you please leave the `FromCancellation` method in under `#ifndef FEATURE_CORECLR`